stable/appmesh-gateway/templates/gateway.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.appmesh.gateway }}
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: {{ template "appmesh-gateway.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "appmesh-gateway.labels" . | indent 4 }}
+spec:
+  namespaceSelector:
+    matchLabels:
+      appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "appmesh-gateway.fullname" . }}
+  listeners:
+    - portMapping:
+        port: 8088
```

Comment:
Should we make the port configurable?

Reply:
Why would a user change it? The ports that matter in my opinion are the ones exposed via NLB.

Reply:
The idea was for users who would like to play with different knobs exposed. Not needed as such